keycloak/helm/keycloak/values.yaml.tpl

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
My guess is you guys need to ask for at least the keycloak fqdn and configure the dependency keycloak chart here.